guides/source/layout.html.erb

```diff
@@ -14,6 +14,15 @@
   <script src="javascripts/turbolinks.js" data-turbolinks-track="reload"></script>
   <script src="javascripts/guides.js" data-turbolinks-track="reload"></script>
   <script src="javascripts/responsive-tables.js" data-turbolinks-track="reload"></script>
+  <meta property="og:title" content="<%= yield(:page_title) || 'Ruby on Rails Guides' %>" />
+  <% if content_for?(:description) %>
+  <meta name="description" content="<%= yield(:description) || 'Ruby on Rails Guides' %>" />
+  <meta property="og:description" content="<%= yield(:description) || 'Ruby on Rails Guides' %>" />
```

Comment:
Shouldn't we remove `if content_for?(:description)` and change these lines to?
```diff
-<meta name="description" content="<%= yield(:description) || 'Ruby on Rails Guides' %>" />
-<meta property="og:description" content="<%= yield(:description) || 'Ruby on Rails Guides' %>" />
+<meta name="description" content="<%= yield(:description).presence || 'Ruby on Rails Guides' %>" />
+<meta property="og:description" content="<%= yield(:description).presence || 'Ruby on Rails Guides' %>" />
```

Reply:
Oops, I forgot removing `|| 'Ruby on Rails Guides'`.😱 

Okay, let's use `.prensece || "xxx"`!